middleware.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @next/next/no-server-import-in-page */
```

Comment:
For some reason eslint complains both if I add `pages/_middleware` AND if I import this here. This is the way it seems to work, but eslint still complains. Let me know if you know how I can fix this.